src/index.js

```diff
@@ -55,14 +64,26 @@ export default class LazilyRender extends React.Component<LazilyRenderProps, Laz
     return convertOffsetToBounds(offset);
   }
 
-  startListening() {
-    const container = this.container;
+  componentDidUpdate(prevProps: ?LazilyRenderProps) {
```

Comment:
I think it's worth adding a unit test in where we setProps with enzyme and check that we remove/add the listeners. Also, to probe that we only care about `scrollContainer` changes. What do you think?

Reply:
That makes a lot of sense to me 😄Have added one in now!